qemu/tests/cfg/nested_test.cfg

```diff
@@ -11,7 +11,8 @@
     type = nested_test
     test_type = testcase
     install_node = no
-    test_timeout = 3600
+    login_timeout = 360
+    test_timeout = 5400
```

Comment:
Suggest set the `test_timeout` for different test.

Reply:
Add test_timeout to test cases separately.